src/topoflow_glacier/bmi/bmi_topoflow_glacier.py

```diff
@@ -889,8 +999,14 @@ def update_dew_point(self):
         b = 18.678
         c = 257.14  # [deg C]
         # d = 234.5    # [deg C]
-        log_term = np.log(self.e_air / a)
-        self.T_dew = c * log_term / (b - log_term)  # [deg C]
+        # Floor e_air to a tiny positive number to avoid -inf/NaN on first step
+        e_air_mbar = np.asarray(self.e_air, dtype="float64")
+        e_air_mbar = np.maximum(e_air_mbar, 1e-9)
```

Comment:
I don't think max() is needed since the next line is a log function and makes all negative values to a positive value